bucket/krew.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "0.4.2",
+    "description": "Krew is the package manager for kubectl plugins.",
+    "homepage": "https://krew.sigs.k8s.io/",
+    "license": "Apache-2.0",
+    "notes": "This script needs to be executed as an administrator.",
+    "suggest": {
+        "gsudo": "gsudo"
+    },
```

```suggestion
    "notes": "This script needs to be executed as an administrator. You can install 'gsudo' to run it.",
```

It's only required during initial setup